backend/routes/task.routes.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express'
+import { TaskController } from "../controllers/task.controller";
+import { authorize, UserType } from '../middleware/credentials.middleware';
+
+const router = express.Router();
+const taskController = new TaskController();
+
+router.get('/one',authorize([UserType.ADMIN]), taskController.getOne.bind(taskController))
```

Comment:
Took a look here and realized you have this `/one` endpoint which is a GET but it expects a body which would actually contain the id of the task. Generally if we make a GET it's not intuitive to pass a body to it although it might work 🧐 would you think it would be better to have a **path param** here with the ID of the task itself?

some docs on Express path params/route params: https://expressjs.com/en/guide/routing.html (<kbd>ctrl</kbd> + <kbd>f</kbd> for `Route parameters`)